Remotion/Web/UnitTests/Core/ExecutionEngine/WxeContextTest.cs

```diff
@@ -39,9 +41,12 @@ public class WxeContextTest
   private string _functionTypeName;
   private string _resource;
 
+  private ServiceLocatorScope _serviceLocatorScope;
+
   [SetUp]
   public virtual void SetUp ()
   {
+    _serviceLocatorScope = WebConfigurationFactory.GetBasicWxeUrlSettings();
```

Comment:
We discussed supplying the WxeSettings via the WxeContext-ctor to avoid having to deal with this. Is there a need/reason we can't do it this way?

Reply:
All methods on wxecontext that actually have the safe service locator are static or use the static methods with the safeservicelocator.

Reply:
Oops. okay.

Reply:
Yea, I also thought we could do something, but the changes got out of hand very quickly :(